compiler/rustc_passes/src/check_attr.rs

```diff
@@ -287,8 +287,9 @@ impl CheckAttrVisitor<'tcx> {
                             self.doc_alias_str_error(meta);
                             return false;
                         }
-                        if let Some(c) =
-                            doc_alias.chars().find(|&c| c == '"' || c == '\'' || c.is_whitespace())
+                        if let Some(c) = doc_alias
+                            .chars()
+                            .find(|&c| c == '"' || c == '\'' || (c.is_whitespace() && c != ' '))
```

Comment:
Why not change it to `is_ascii_whitespace`?

Reply:
Because I assume it includes `\t`, `\r` and `\n`, which we still don't want.

Reply:
Yes, it is there.
```rust
        match *self {
            '\t' | '\n' | '\x0C' | '\r' | ' ' => true,
            _ => false,
        }
```